.rubocop.yml

```diff
@@ -1013,6 +1031,9 @@ Rails/WhereNot:
 Rails/WhereNotWithMultipleConditions:
   Enabled: true
 
+Rails/WhereRange:
+  Enabled: false
```

Comment:
I think we _could_ enable this. It's a bit more of a stylistic one, and the "Safety" notes gave me pause to trust the autocorrecter without closer inspection.

https://docs.rubocop.org/rubocop-rails/cops_rails.html#railswhererange

Reply:
Yeah, it seems more stylistic and I don't find it significantly better? I could go either way on it.

Reply:
Personally I have trouble recalling whether `..` or `...` is the inclusive version in a range (i.e. `<=` or `>=`), so I'd worry this could also negatively impact readability. Maybe that's just me though 😄